src/index.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+const port = 8000;
+
+
+app.get('/', (req, res) => {
+
+
+    const htmlPath = path.join(__dirname, 'index.html');
+
+    res.sendFile(htmlPath);
+});
+
+class MyServer {
+    constructor (currentPort) {
+        this.server = null;
+        this.port = currentPort;
+    }
+
+    async startServer () {
+        const promise = new Promise(resolve => {
```

Comment:
Let's use [events.once](https://nodejs.org/api/events.html#events_events_once_emitter_name) instead of Promise constructor and `try/catch` + `async/await` to handle exceptions.